### PR TITLE
Upgrade to gulp-sass 5.0.0 to use node-sass 6.0.0

### PR DESCRIPTION
**Pull request by prameshj**
https://github.com/dlmanning/gulp-sass/blob/978b8f6b48a3d4da1110076b5a246a5e80f7477e/package.json#L46 uses node-sass 6.0.0 which should support Node 12.

### CODE REVIEW

**Comment by prameshj** [comment]
`npm install firebase --save` worked after changes in this PR.

In order to run the demo, I also had to run:

```
npm cache clear --force
npm install sass
npm install
npm run demo
```

I verified "sign in with phone", "sign in with email password" and "continue as guest" flows in the demo app UI with my changes and they worked.

File: package.json [hunk 1]
```diff
@@ -53,15 +53,16 @@
...
     "gulp-util": "^3.0.7",
     "protractor": "^7.0.0",
     "streamqueue": "^1.1.1",
     "webdriver-manager": "^12.1.8"
   },
   "dependencies": {
     "dialog-polyfill": "^0.4.7",
-    "material-design-lite": "^1.2.0"
+    "material-design-lite": "^1.2.0",
+    "sass": "^1.49.7"
```
Comment (bojeil-google) [inline package.json:65]: Is this needed here? I assume this is only needed for the build process (in gulpfile only). So only `devDependencies` need to be updated?

Comment (prameshj) [reply package.json:65]: These got autogenerated when i ran `npm run demo`. I tried reverting, but they regenerated, so added them in the PR.

Comment (bojeil-google) [reply package.json:65]: I think `npm install sass` i adding this. You should run `npm install sass --save-dev`

Comment (prameshj) [reply package.json]: You are right. Removed this line and ran`npm install sass --save-dev` to run `npm install` successfully.

**Review by bojeil-google — APPROVED** [review]

**Comment by prameshj** [comment]
This should fix - https://github.com/firebase/firebaseui-web/issues/716

**Pull request merged**